src/components/molecules/PollBox/PollBox.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useCallback, useMemo } from "react";
+import { useForm, useFieldArray } from "react-hook-form";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPaperPlane, faPlus } from "@fortawesome/free-solid-svg-icons";
+
+import { MAX_POLL_CHOICES } from "settings";
+
+import { getVenueOwners } from "api/admin";
+
+import { PollValues, PollQuestion } from "types/chat";
+
+import { InputField } from "components/atoms/InputField";
+
+import { useVenueId } from "hooks/useVenueId";
+import { useRoles } from "hooks/useRoles";
+import { useUser } from "hooks/useUser";
+
+import "./PollBox.scss";
+
+export interface PollBoxProps {
+  createPoll: (props: PollValues) => void;
+}
+
+const DEFAULT_QUESTION: Partial<PollQuestion> = { name: "" };
+const DEFAULT_VALUES = {
+  topic: "",
+  questions: [DEFAULT_QUESTION, DEFAULT_QUESTION],
+};
+
+export const PollBox: React.FC<PollBoxProps> = ({ createPoll }) => {
+  const venueId = useVenueId();
+  const { userRoles } = useRoles();
+  const { user } = useUser();
```

Comment:
If you need only id, use userId

Reply:
fixed in f2fa24aadcae7547799e463bd593e8a7a2cf05b5